function-support.html

```diff
@@ -947,6 +931,21 @@ <h2 id="annotation">Annotation</h2>
 		</tr>
 	</table>
 	
+	<div class="indented">
+		<p><code>\tag{hi} x+y^{2x}</code></p>
+		<p>\[\tag{hi} x+y^{2x} \]</p>
+		<p><code>\tag*{hi} x+y^{2x}</code></p>
+		<p>\[\tag*{hi} x+y^{2x} \]</p>
+		<p><code>\tag</code> is not yet supported inside the <code>{gather}</code> environment.</p>
+	</div>	
+	
+<h3 id="line-breaks">Line Breaks</h3>
+	<p>KaTeX 0.10.0+ will insert automatic line breaks in inline math. These can be suppressed by placing math inside a pair of braces, as in <code>{F=ma}</code>.</p>
+	
+	<p>Hard line breaks are <code>\\</code> and <code>\newline</code>.</p>
+	
+	<p>In display math, KaTeX does not insert automatic line breaks. It ignores display math hard line breaks when rendering option <code>strict: true</code>.</p>

```

Comment:
Maybe add "to match LaTeX"?  Maybe this is getting too redundant though.  Feel free to ignore this comment in particular.